data-collections-exercises.md

```diff
@@ -9,89 +9,147 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+planeteers[1]
+=> "Wind"
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+planeteers.push"Heart"
+=> ["Earth", "Wind", "Captain Planet", "Fire", "Water", "Heart"]
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers.delete_at(2)
+=> "Captain Planet"
```

Comment:
you used a method we need you to know the simple cool trick that Ruby provide which is the subtraction `planeteers. - ["Captain Planet"]`

Reply:
Nice! much easier. Thank you

Reply:
you're welcome ⭐️